staging/traefik2/README.md

```diff
@@ -0,0 +1,15 @@
+# Traefik
+
+[Traefik](https://traefik.io/) is a modern HTTP reverse proxy and load balancer made to deploy
+microservices with ease.
+
+**NOTE: THIS CHART IS IN INCUBATOR STATUS, DO NOT USE IN A PRODUCTION ENVIRONMENT**
```

Comment:
scary note

Reply:
No addons will be created from this today. @joejulian wants to test this out. I am also using this chart as a test bed for a few things.